ui/src/app/controls/multi-command/multi-command.component.html

```diff
@@ -3,13 +3,15 @@ <h3>{{title}}</h3>
     <span *ngFor="let control of form.controls; index as i">
         <mat-form-field appearance="fill">
             <mat-label><span>Command</span></mat-label>
-            <mat-select [formControl]="control">
+            <mat-select [attr.data-cy]="'command-selector-'+i" [formControl]="control">
                 <mat-option *ngFor="let commandElement of commandList" [value]="commandElement">{{commandElement}}</mat-option>
             </mat-select>
         </mat-form-field>
+        <button [attr.data-cy]="'command-minus-'+i"  mat-icon-button (click)="removeCommand(i)">
+            <mat-icon class="tab-icon material-icons-outlined">remove</mat-icon>
+        </button>
     </span>
-    <button *ngIf="form.controls.length > 0" mat-icon-button (click)="addCommand('')">
-        <mat-icon class="tab-icon material-icons-outlined">add</mat-icon>
-    </button>                
-    <button *ngIf="form.controls.length == 0" mat-flat-button (click)="addCommand('')">{{addLabel}}</button>
+    <div>
```

Comment:
I don't know exactly why, the "Add command" button is not displayed at the same place:
- above when editing a container with existing commands
![command1](https://github.com/redhat-developer/odo/assets/9973512/8f5f9269-8735-4356-abd6-ad6732ccebf7)

- below when creating a new container, when adding commands
![command2](https://github.com/redhat-developer/odo/assets/9973512/ddbcfd6f-4e44-4d44-8228-34228737e900)

Reply:
🤔 Indeed, good catch. That's weird. I'll take a look.

Reply:
Ah, I think the issue is that there were no `command` in the Devfile, only `args` set to `tail -f /dev/null`. At least, that's what I saw in the `go` and `nodejs` Devfiles coming from the default registry:

```yaml
- container:
    args:
    - tail
    - -f
    - /dev/null
    endpoints:
    - name: http-go
      targetPort: 8080
    image: registry.access.redhat.com/ubi9/go-toolset:1.18.10-4
    memoryLimit: 1024Mi
    mountSources: true
  name: runtime
```

To make it more explicit, I've just tried to surround each type in a dedicated card, like so:
![image](https://github.com/redhat-developer/odo/assets/593208/dca28e32-873b-4a7a-9cdc-4a0b4deac2ff)

![image](https://github.com/redhat-developer/odo/assets/593208/363f6485-2fd6-4ebe-b192-741b00329253)


Hopefully, it makes things clearer.. Let me know what you think.